app/controllers/sign_up/completions_controller.rb

```diff
@@ -0,0 +1,24 @@
+module SignUp
+  class CompletionsController < ApplicationController
```

Comment:
Given that this page should only be displayed under one of the following 2 conditions, I would expect the preconditions to be as follows:

1. The user has finished creating their account for the first time and did it by coming via an SP

2. The user has finished proofing for the first time. Note that there are really 2 scenarios here: one with an SP, one without since we allow proofing without coming from an SP. In the case without the SP, the link to continue should point to the profile page as opposed to an SP.